tests/unit/multipass/test_multipass.py

```diff
@@ -728,9 +729,10 @@ def test_version_decode_error(fake_process):
         ["multipass", "version"], stdout=b"multipass 1.6.2\xd0"
     )
 
-    with mock.patch("locale.getpreferredencoding", return_value="ascii"):
-        with pytest.raises(MultipassError) as exc_info:
-            Multipass().version()
+    with mock.patch("locale.getpreferredencoding", return_value="ascii"), pytest.raises(
```

Comment:
OK I especially don't like SIM117 in tests here, since the two context managers are conceptually quite distinct in their purposes (unlike in the production code where they're related).

Having seen this I think I have a preference for ignoring SIM117 for tests at least.

Reply:
From our offline conversation, we agreed that combining `with` statements and having an 88 character line limit makes the code less readable.  

I undid the SIM117 changes, ignored the rule, and pushed the change to starbase. https://github.com/canonical/starbase/pull/90